src/main/java/bookstore/service/impl/BookServiceImpl.java

```diff
@@ -38,4 +38,19 @@ public List<BookDto> findAll() {
                 .map(bookMapper::toDto)
                 .toList();
```

Comment:
Mapstruct supports mapping collections from the box

Reply:
@MAMentorFX so, i can just "return bookRepository.findAll()" if I get it right ?

Reply:
please move this mapping logic to the controller level. And also you didn't fix an issue mentioned by the previous mentor